actionmailer/README.rdoc

```diff
@@ -128,7 +128,7 @@ Action Mailer is released under the MIT license:
 
 API documentation is at
 
-* http://api.rubyonrails.org
+* https://api.rubyonrails.org/
```

Comment:
Why add the trailing slash to `actionmailer/README.rdoc` and `actioncable/README.md` unlike others?